.travis.yml

```diff
@@ -6,8 +6,8 @@ before_install:
   PATH=`echo $PATH | sed "s/\.\/node_modules\/\.bin//g"`;
   export PATH=$PATH:./node_modules/.bin;
   touch .env;
-  rvm install ruby-2.3.0 --binary && rvm use ruby-2.3.0 && bundle install;
-script: npm run test:all
+  if [ $IS_INTEGRATION_TEST ]; then rvm install ruby-2.3.0 --binary && rvm use ruby-2.3.0 && bundle install; else echo "Not an integration test. Skipping Ruby setup"; fi
```

Comment:
If there's a better way to do this in bash (skip the ruby setup for non-integration tests), let me know.